exporter/splunkhecexporter/hec_worker.go

```diff
@@ -27,7 +28,10 @@ type defaultHecWorker struct {
 }
 
 func (hec *defaultHecWorker) send(ctx context.Context, buf buffer, headers map[string]string) error {
-	req, err := http.NewRequestWithContext(ctx, "POST", hec.url.String(), buf)
+	nb := make([]byte, buf.Len())
+	copy(nb, buf.Bytes())
```

Comment:
Can you please put a comment here that we are doing it to work around https://github.com/golang/go/issues/51907 ?

Reply:
done thanks!